README.md

```diff
@@ -4,58 +4,66 @@ This is a lightweight image writing app with a simple four-step GUI for creating
 
 ## Translation Changes
 
-Changes to master language (en_GB) strings can be submitted via pull request to this GitHub repo. English strings are periodically synchronised to a Transifex project: https://www.transifex.com/libreelec/libreelec-usb-sd-creator allowing contributors to translate them into other languages. Translated strings are periodically synchronised from Transifex back to this repo for inclusion in the next release.
+Changes to master language (en_GB) strings can be submitted via pull request to this GitHub repo. English strings are periodically synchronised to a Transifex project: <https://www.transifex.com/libreelec/libreelec-usb-sd-creator> allowing contributors to translate them into other languages. Translated strings are periodically synchronised from Transifex back to this repo for inclusion in the next release.
```

Comment:
better place link on text

Reply:
`<http...>` is valid markdown for just links, current formatting is not

Reply:
I mean, link embedded in text would be more readable in this case

FYI for GFMD angle brackets aren't needed: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls#urls